### PR TITLE
Add gitlab documentation

### PR DESCRIPTION
**Pull request by OriPekelman**
I verified the integration to work, this is not yet GA as not deployed on EU, US and AU. Please verify last paragraph to be true with Gitlab.

### CODE REVIEW

File: src/administration/integrations/gitlab.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Optional parameters:
+* `fetch-branches`: Track and deploy branches (true by default)
+* `build-merge-requests`: Track and deploy merge-requests (true by default)
+* `merge-requests-clone-parent-data` : should merge requests clone the data from the parent environment
+
+### 3. Add the webhook
+
+The previous command, if succesful should output the configuration of the integration. The last element would look like
+```| hook_url                         | https://{region}.platform.sh/api/projects/{projectid}/integrations/{hook_id}/hook |`
```
Comment (Crell) [inline src/administration/integrations/gitlab.md]: This looks like a formatting error?

The ``` should be on their own line and specify what the highlighting is.  And it looks like it's just a single ` at the end of the line, which is likely wrong.

File: src/administration/integrations/gitlab.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Open a terminal window (you need to have the Platform.sh CLI installed). Enable the GitLab integration as follows:
+
+```bash
+platform integration:add --type=gitlab --token=THE-TOKEN-YOU-WROTE-DOWN --base-url=https://THE-URL-OF-YOUR-GITLAB/ --gitlab-project=MY-NAMESPACE/MY-PROJECTNAME --merge-requests-clone-parent-data=true --build-merge-requests=true --fetch-branches=true
+```
+
+Optional parameters:
+* `fetch-branches`: Track and deploy branches (true by default)
+* `build-merge-requests`: Track and deploy merge-requests (true by default)
+* `merge-requests-clone-parent-data` : should merge requests clone the data from the parent environment
```
Comment (Crell) [inline src/administration/integrations/gitlab.md]: Specify the default.  (I believe it's true.)

File: src/administration/integrations/gitlab.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### 1. Generate a token
+
+To integrate your Platform.sh project with an existing GitLab
+repository, you first need to generate a token on your GitLab user
+profile. Simply go to your Settings page on GitLab and click
+`Access Tokens`.
+
+Fill the `Name` field for example with "Platform.sh Integration" and optionally
+set an expiration time.
```
Comment (Crell) [inline src/administration/integrations/gitlab.md]: Here and above should use soft word wrapping.  (Probably a copy-paste from another page that we haven't gotten around to fixing yet.)

File: src/administration/integrations/gitlab.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+```bash
+platform integration:add --type=gitlab --token=THE-TOKEN-YOU-WROTE-DOWN --base-url=https://THE-URL-OF-YOUR-GITLAB/ --gitlab-project=MY-NAMESPACE/MY-PROJECTNAME
+```
+
+Optional parameters:
+* `--fetch-branches`: Track and deploy branches (true by default)
+* `--build-merge-requests`: Track and deploy merge-requests (true by default)
+* `--merge-requests-clone-parent-data` : should merge requests clone the data from the parent environment (true by default)
+
+### 3. Add the webhook
```
Comment (pjcdawkins) [inline src/administration/integrations/gitlab.md]: We don't add the hook automatically?

Comment (OriPekelman) [reply src/administration/integrations/gitlab.md]: I don't believe so. If we can that would be a very good thing (also for github etc.. there is a ticket for that)

File: src/administration/integrations/gitlab.md [hunk 5]
```diff
@@ -49,13 +49,16 @@ Optional parameters:
...
-Now, copy the hook URL 
+```
+| hook_url | https://{region}.platform.sh/api/projects/{projectid}/integrations/{hook_id}/hook |
+```
+
+Now, copy the hook URL.
 
-Go to your GitLab repository and click `Settings > Integrations` and paste the Payload URL, in the Triggers secrion choose 
-`Push events`, `Tag push events` and `Merge Request events`. And click on `Add bebhook`.
+Go to your GitLab repository and click `Settings > Integrations` and paste the Payload URL, in the Triggers section choose
```
Comment (OriPekelman) [inline src/administration/integrations/gitlab.md]: @pjcdawkins ++ for catching my typos :) sorry for that

**Comment by kaystrobach** [comment]
Will you set the build status in gitlab via the api?
Sry just using EU region, no chance to test yet

**Comment by OriPekelman** [comment]
Real soon. We will ping you as soon as it is available.

**Comment by kaystrobach** [comment]
Aaawweeesoome 👍🤪🤩

I hope to start a new project in the next days which used platform.sh via gitlab ... 😄

**Pull request merged**